DevCycleTests/Models/DVCClientTest.swift

```diff
@@ -170,7 +172,67 @@ class DVCClientTest: XCTestCase {
         client.refetchConfig(sse: true, lastModified: 789)
         XCTAssertEqual(service.numberOfConfigCalls, 6)
     }
+    
+    func testSseCloseGetsCalledWhenBackgrounded() {
+        let service = MockService()
+        let user1 = try! DVCUser.builder().userId("user1").build()
+        let client = try! DVCClient.builder().user(user1).environmentKey("my_env_key").build(onInitialized: nil)
+        client.setup(service: service)
```

Comment:
this call to setup doesn't mock out all the calls properly, thats why it has to be sent through the builder